airflow/models/taskinstance.py

```diff
@@ -1871,3 +1872,14 @@ def construct_task_instance(self, session=None, lock_for_update=False) -> TaskIn
         else:
             ti = qry.first()
         return ti
+
+
+STATICA_HACK = True
+globals()['kcah_acitats'[::-1].upper()] = False
+if STATICA_HACK:  # pragma: no cover
+    from sqlalchemy.orm import relationship
+
+    from ..job import base_job
+    from . import dagrun
+    TaskInstance.dag_run = relationship(dagrun.DagRun)
+    TaskInstance.queued_by_job = relationship(base_job.BaseJob)
```

Comment:
Can we add a comment on what this is for

Reply:
Oh yes. To let pylint know about the relationship created without creating an import cycle